gui/system/forms.py

```diff
@@ -2608,7 +2612,11 @@ class CertificateImportForm(ModelForm):
         label=models.Certificate._meta.get_field('cert_certificate').verbose_name,
         widget=forms.Textarea(),
         required=True,
-        help_text=models.Certificate._meta.get_field('cert_certificate').help_text
+        help_text=_(
+            "Cut and paste the contents of your certificate here.<br>"
+            "Order in which you should paste this: <br>"
+            "The Primary Certificate.<br>The Intermediate Certificate(s) (optional)."
```

Comment:
Root Certificate Authority's Cert and Intermediate Certificate Authority's Cert is what you want to write here and not just inter cert and root cert. You may resort to CA as a short form if need be (instead of Certitificate Authority)